src/vs/workbench/services/editor/common/editorGroupsService.ts

```diff
@@ -418,6 +418,7 @@ export interface IGroupChangeEvent {
 	kind: GroupChangeKind;
 	editor?: EditorInput;
 	editorIndex?: number;
+	previousEditorIndex?: number;
```

Comment:
Clarify in comment when this applies and what it is? Also a test would be nice for ensuring the properties are correct when a move event occurs.